From 62583ddcfcd13e60ed68b258ed5e9e14cdae873c Mon Sep 17 00:00:00 2001
From: hyphop <art@khadas.com>
Date: Fri, 24 Apr 2020 21:53:59 +0900
Subject: [PATCH 23/53] add hostname as CONFIG_IDENT_STRING for net boot

add hostname value same as CONFIG_DEFAULT_DEVICE_TREE
hostname value used as dhcp host name
---
 include/configs/meson64.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/configs/meson64.h b/include/configs/meson64.h
index 50707a31..87ffc7c9 100644
--- a/include/configs/meson64.h
+++ b/include/configs/meson64.h
@@ -89,6 +89,9 @@
 	BOOTENV
 #endif
 
+#define CONFIG_HOSTNAME CONFIG_DEFAULT_DEVICE_TREE
+#define CONFIG_BOOTP_SEND_HOSTNAME 1
+
 #include <config_distro_bootcmd.h>
 
 #endif /* __MESON64_CONFIG_H */
-- 
2.17.1

